### PR TITLE
Add basic scala stuff

### PR DESCRIPTION
**Pull request by blorente**
I hope I haven't messed anything up

### CODE REVIEW

File: roles/scala/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  group: 
+    name: scala
+    state: present
+
+- name: create /var/lib/scala (scala)
+  file:
+    path: /var/lib/scala
+    group: scala
+    recurse: yes
+    mode: "g+rwx"
```
Comment (fikovnik) [inline roles/scala/tasks/main.yml]: Should not this be `g+ws` ?

File: roles/scala/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    state: directory
+
+- name: add oracle repo (scala)
```
Comment (fikovnik) [inline roles/scala/tasks/main.yml]: Do we need oracle java? I forgot what was running on the old prl3, but I had an impression that it was openjdk

Comment (blorente) [reply roles/scala/tasks/main.yml]: Nope, we used oracle java

**Comment by blorente** [comment]
Should be fixed now

**Pull request merged**